src/App/Logger/Error/Reader.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\App\Logger\Error;
+
+use Magento\MagentoCloud\Filesystem\FileList;
+use Magento\MagentoCloud\Package\UndefinedPackageException;
+
+/**
+ * @inheritDoc
+ */
+class Reader implements ReaderInterface
+{
+    /**
+     * @var FileList
+     */
+    private $fileList;
+
+    /**
+     * @param FileList $fileList
+     */
+    public function __construct(FileList $fileList)
+    {
+        $this->fileList = $fileList;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function read(): array
+    {
+        try {
+            $handle = @fopen($this->fileList->getCloudErrorLog(), 'r');
```

Comment:
why do you ignore error here?

Reply:
Because we can't throw an exception and stop the process in a case when `cloud.error.log` can't be read. It's not critical for the deployment process.